test/other/codesize/test_codesize_cxx_mangle.size

```diff
@@ -1 +1 @@
-232600
+232604
```

Comment:
This rebase just landed separately: https://github.com/emscripten-core/emscripten/pull/23918